public/app/features/live/pipeline/models.gen.ts

```diff
@@ -0,0 +1,116 @@
+/* Do not change, this code is generated from Golang structs */
+
+import { FieldConfig } from '@grafana/data';
+
+export interface JsonFrameConverterConfig {}
+export interface AutoInfluxConverterConfig {
+  frameFormat: string;
+}
+export interface ExactJsonConverterConfig {
+  fields: Field[];
+}
+export interface Label {
+  name: string;
+  value: string;
+}
+export interface Field {
+  name: string;
+  type: number;
+  value: string;
+  labels?: Label[];
+  config?: FieldConfig;
+}
+export interface AutoJsonConverterConfig {
+  fieldTips?: { [key: string]: Field };
+}
+export interface ConverterConfig {
+  type: Omit<keyof ConverterConfig, 'type'>;
+  jsonAuto?: AutoJsonConverterConfig;
+  jsonExact?: ExactJsonConverterConfig;
+  influxAuto?: AutoInfluxConverterConfig;
+  jsonFrame?: JsonFrameConverterConfig;
+}
+export interface MultipleFrameProcessorConfig {
+  processors: FrameProcessorConfig[];
+}
+export interface KeepFieldsFrameProcessorConfig {
+  fieldNames: string[];
+}
+export interface DropFieldsFrameProcessorConfig {
+  fieldNames: string[];
+}
+export interface FrameProcessorConfig {
+  type: Omit<keyof FrameProcessorConfig, 'type'>;
+  dropFields?: DropFieldsFrameProcessorConfig;
+  keepFields?: KeepFieldsFrameProcessorConfig;
+  multiple?: MultipleFrameProcessorConfig;
+}
+
+export interface ChangeLogOutputConfig {
+  fieldName: string;
+  channel: string;
+}
+export interface LokiOutputConfig {
+  uid: string;
+}
+export interface RemoteWriteOutputConfig {
+  uid: string;
+  sampleMilliseconds: number;
+}
+export interface ThresholdOutputConfig {
+  fieldName: string;
+  channel: string;
+}
+export interface NumberCompareFrameConditionConfig {
+  fieldName: string;
+  op: string;
```

Comment:
Would it be possible to generate the underlying enum as well? https://github.com/grafana/grafana/blob/5358c45a3ade9420bc014e2bc0ef91f971ce0dbb/pkg/services/live/pipeline/compare.go#L4

Reply:
I am still convinced we will want to end up with this generated from cue -- for frontend panels we are struggling to get this all working perfectly, but when it does it will be :fire: 

https://github.com/grafana/cuetsy/issues/35


Lets start with the types we have defined here and add more and fixes as necessary

Reply:
Generating enums will simplify UX drastically. But not possible with the tool we are using for the current iteration - so let's leave for the cuetsy